docker/ingest/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN apt-get update && \
 
 
 # The builder image, used for building the virtual environment
-FROM python:3.11-slim-bookworm AS builder
+FROM python:3.12-slim-bookworm AS builder
```

Comment:
We have a number of layers in this Dockerfile so this will also need to be done on Line 6 & Line 72.